### PR TITLE
Mute/Unmute commands and button

### PR DESCRIPTION
**Pull request by marceloschmidt**
This closes https://github.com/RocketChat/Rocket.Chat/issues/214

### CODE REVIEW

File: packages/rocketchat-lib/server/methods/sendMessage.coffee [hunk 1]
```diff
@@ -15,6 +15,17 @@ Meteor.methods
 		if not room
 			return false
 
+		subscription = RocketChat.models.Subscriptions.findOneByRoomIdAndUserId room._id, user._id
+		console.log subscription
```
Comment (rodrigok) [inline packages/rocketchat-lib/server/methods/sendMessage.coffee]: Do you need this log?

Comment (marceloschmidt) [inline packages/rocketchat-lib/server/methods/sendMessage.coffee]: No! I'll remove it

**Comment by engelgabriel** [comment]
What happens if the user is renamed?

**Comment by engelgabriel** [comment]
lgtm

**Comment by rodrigok** [comment]
LGTM

**Comment by graywolf336** [comment]
Curious to know the answer to @engelgabriel's question..

**Comment by engelgabriel** [comment]
It updated the references too

**Pull request merged**